multimodal/src/autogluon/multimodal/data/datamodule.py

```diff
@@ -58,6 +59,10 @@ def __init__(
         id_mappings
              Id-to-content mappings. The contents can be text, image, etc.
              This is used when the dataframe contains the query/response indexes instead of their contents.
+        val_is_train
+             Whether we want to treat validation as training.
```

Comment:
I think we should say `whether we are triggering `is_training` when creating the dataset for validation`. We can make the docstring more clear.

Reply:
Modified the docstring.